packages/appeals-service-api/src/controllers/appeals.js

```diff
@@ -0,0 +1,27 @@
+const { getAppealsByEmail } = require('../services/appeal.service');
+const logger = require('../lib/logger');
+const ApiError = require('#errors/apiError');
+
+const getUserAppealsByEmail = async (req, res) => {
```

Comment:
think this should probably be a new v2 endpoint, that checks for linked appeals in sql rather than mongo